app/views/forums/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<p style="color: green"><%= notice %></p>
+
+<h1>Forums</h1>
+
+<div id="forums">
+  <% @forums.each do |forum| %>
+    <%= render forum %>
+    <p>
+    <strong>Forum name:</strong>
```

Comment:
remove line 9 -15 as you already have this info on your _forum.html.erb that you are rendering